tool/utils.ts

```diff
@@ -118,16 +119,17 @@ export async function getDartSassEmbedded(options: {
   const repo = 'dart-sass-embedded';
 
   if (options.release) {
-    const latestRelease = await getLatestReleaseInfo({
-      repo,
-      // TODO(awjin): versionConstraint
-    });
+    const release = options.version
+      ? {tag_name: options.version, name: `sass_embedded ${options.version}`}
+      : await getLatestReleaseInfo({
+          repo,
```

Comment:
```suggestion
          repo
```

Nit: this'll save a few vertical lines after formatting :smiley:

Reply:
I'd like that too but the formatter is enforcing this comma's existence (probably because these are function arguments?)